CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 2.14.0 (2018-11-29)
+- Added `stache-hide-from-search` component to allow users to remove content from Stache Search results. [#557](https://github.com/blackbaud/stache2/pull/557)
```

Comment:
In the Component docs, we don't format component names as code snippets. It probably doesn't matter here in the changelog, but if "Added hide from search component..." works just as well, it would be more consistent with the formatting in the Component docs.

Not sure if it matters, but should "remove" be "exclude"? I mean, are we excluding the content from the search altogether, or are we including it in the search and then removing it from the search results? I assume the former. If so, "exclude" might be the better word choice here.

Also, "search" at the end should not be capitalized.

Reply:
Actually, disregard the first point about the code snippet. It's not how we format component names in the docs, but it's consistent with the rest of the changelog. We might want to get those on the same page at some point, but we should be consistent within the changelog for now.